src/Docker/Runner.php

```diff
@@ -37,80 +38,44 @@ class Runner
 
     const MODE_RUN = 'run';
 
-    /**
-     * @var ObjectEncryptorFactory
-     */
-    private $encryptorFactory;
-
-    /**
-     * @var ClientWrapper
-     */
-    private $clientWrapper;
-
-    /**
-     * @var Credentials
-     */
-    private $oauthClient;
-
-    /**
-     * @var Credentials
-     */
-    private $oauthClient3;
-
-    /**
-     * @var LoggersService
-     */
-    private $loggersService;
-
-    /**
-     * @var string
-     */
-    private $commandToGetHostIp;
+    private ObjectEncryptorFactory $encryptorFactory;
+    private ClientWrapper $clientWrapper;
+    private Credentials $oauthClient;
+    private Credentials $oauthClient3;
+    private SandboxesApiClient $sandboxesApiClient;
+    private LoggersService $loggersService;
+    private string $commandToGetHostIp;
+    private int $minLogPort;
+    private int $maxLogPort;
+    private array $instanceLimits;
 
-    /**
-     * @var int
-     */
-    private $minLogPort;
-
-    /**
-     * @var int
-     */
-    private $maxLogPort;
-
-    /**
-     * @var array
-     */
-    private $instanceLimits;
-
-    /**
-     * Runner constructor.
-     * @param ObjectEncryptorFactory $encryptorFactory
-     * @param ClientWrapper $clientWrapper
-     * @param LoggersService $loggersService
-     * @param string $oauthApiUrl
-     * @param array $instanceLimits
-     * @param int $minLogPort
-     * @param int $maxLogPort
-     */
     public function __construct(
         ObjectEncryptorFactory $encryptorFactory,
         ClientWrapper $clientWrapper,
         LoggersService $loggersService,
-        $oauthApiUrl,
+        string $oauthApiUrl,
         array $instanceLimits,
-        $minLogPort = 12202,
-        $maxLogPort = 13202
+        int $minLogPort = 12202,
+        int $maxLogPort = 13202
```

Comment:
don't we still need to support php 5.6?

Reply:
This is for new Job Queue only which is all PHP 7.4, so no :) I hope we don't have to backport this to old queue :smile: